filename.go

```diff
@@ -0,0 +1,10 @@
+package slinguist
+
+func GetLanguageByFilename(filename string) (lang string, safe bool) {
+	lang, safe = languagesByFilename[filename]
+	if lang == "" {
+		lang = OtherLanguage
```

Comment:
The role of OtherLanguage is pretty important, maybe we should explain that clearly somewhere.